### ISSUE TITLE
Repo should be in its own account

### ISSUE THREAD

**Issue by jmichelz** [OPSN/MVP-discuss#5]
I wanted to try out this development process:
https://hintjens.gitbooks.io/social-architecture/content/chapter4.html
If the main repo is under one person's account, that implies that they have final say over it. Which is opposite of what the C4 says (Collective Code Construction Contract).

**Comment by oresmus** [comment]
I have no strong feeling about this. 

I think there will be many repos -- this one for discussion and one for each coding project. Many of them will be some person's. Is this the "main one"? Maybe. But my "final say" is limited -- I can't force anyone to contribute, nor prevent anyone from forking.

But as I said privately, feel free to fork this repo into a new github account (not one of our personal ones). But then you did fork, and it turned out the repo's issues were not copied. I looked into this slightly more -- I too see no issues on the new repo (not just an empty issues tab, but a complete lack of any issues tab existing, unless I missed it). Maybe github forks never have their own issues? I looked at a few repos I have forks of, and indeed they too seem to completely lack an issues tab.

Anyway, I think for now, that means it's most efficient to keep working in this one, but as soon as we need to make new repos (for code, or discussions about distinct subprojects) we might as well make them in the new github organization you made (called OPSN). If this repo proves long lived and its being in my account bothers you, we can revisit this later and perhaps just start all new discussions in a new repo. (But I'm suspecting that we might find a better place to keep these discussions than in any github repo's issues database. Not to mention that a "better wiki for discussions" ought to be a reasonably early goal for OPSN software to support.)

**Comment by oresmus** [comment]
I've changed my mind (meaning we now agree) -- moving this repo to the separate github account would be worth the initial trouble. (Reasons: newcomers to the project will be less confused if my unrelated personal repos are not "sibling repos" to this one, especially once there are several OPSN-related repos. And that way, this one can get a more descriptive title, maybe MVP-discuss. Finally, I'm wondering if there is an asymmetry in features, for this repo -- I edited one of your (JM's) issue comments, but I'd be surprised if you can edit mine in my own repo, at least by default.)

Since our attempt to fork this repo failed to copy issues or even to have a place for issues (see above), let's just make a new repo. (I'll do this in a moment.) 

We can manually copy the existing issues so they have the same issue numbers. (There is no need to copy all their comments right away, but before we start _adding_ comments to them, we should either copy the older comments, or summarize them and refer to original repo for more.)

**Comment by oresmus** [comment]
I created that repo and the first 4 issues, and copied over all my initial comments on them (it didn't take very long). If you (JM) agree with this plan, you should create the next two issues so they still correctly say who created them (and add or paraphrase your initial comments to them, and your next runs of comments on each of the first four issues).

The two comment threads I think are worth paraphrasing rather than copying exactly are this one (we can just refer to it here), and the long one about a product being useable (we might as well split out the separate idea we discussed for a long time into its own new issue number 8).

**Comment by oresmus** [comment]
JM and I have now finished copying issues and their comments from that old repo, which should no longer be used. It is now ok to show this new repo (OPSN/MVP-discuss) to other people who might add new issues or comment on old ones. Therefore I am closing this issue.

**Comment by oresmus** [comment]
(Update: the obsolete other repos -- oresmus/OPSN, and (its fork with no issues) OPSN/OPSN -- have now been deleted.)
